src/drc/tests/test_partupload.py

```diff
@@ -0,0 +1,162 @@
+import uuid
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import override_settings
+
+from privates.test import temp_private_root
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.constants import VertrouwelijkheidsAanduiding
+from vng_api_common.tests import (
+    JWTAuthMixin, get_operation_url, get_validation_errors, reverse
+)
+
+from drc.api.tests.utils import split_file
+from drc.datamodel.models import EnkelvoudigInformatieObject
+
+INFORMATIEOBJECTTYPE = 'https://example.com/ztc/api/v1/catalogus/1/informatieobjecttype/1'
+
+
+@temp_private_root()
+@override_settings(LINK_FETCHER='vng_api_common.mocks.link_fetcher_200',
+                   CHUNK_SIZE=10)
+class FileUploadAPITests(JWTAuthMixin, APITestCase):
+
+    heeft_alle_autorisaties = True
+
+    def _create_metadata(self):
```

Comment:
For my understanding: What happens when I update a file's metadata and download the file link? Do I get the old file or an error?